cmd/deploy.go

```diff
@@ -137,6 +139,7 @@ generates a deployment manifest (yaml) file if one is not present, and uses it t
 	}
 
 	deployCmd.PersistentFlags().BoolVar(&config.generate, "generate-only", false, "DEPRECATED - Only generate the deployment configuration file. Do not deploy the project.")
+	deployCmd.PersistentFlags().BoolVar(&config.build, "no-build", true, "Deploys the application without building a new image or modifying the deployment configuration file.")
```

Comment:
Did you test this? I have doubts that it will work. I think flag defaults to `true`, and if it exists, it is set to `true`. So the way I see it can never be `false`.
Probably better to have the variable be `config.nobuild` and the flag default to `false`

Reply:
Revert these changes, good catch!